docs/nx-cloud/reference/nx-cloud-cli.md

```diff
@@ -182,10 +177,10 @@ individual commands as follows:
 {% /tab %}
 {% /tabs %}
 
-## npx nx-cloud stop-all-agents
-
-Same as `npx nx-cloud complete-ci-run`.
```

Comment:
This should stay documented in some way -- tons of users do not touch their pipelines very often and we generated with this command for years.

I would either not touch this, or if we really want to change the text, add something about how `complete-ci-run` should be the preferred command.